api/lib/infrastructure/repositories/membership-repository.js

```diff
@@ -94,70 +82,104 @@ const findAdminsByOrganizationId = async function ({ organizationId }) {
+  queryBuilder.orderByRaw('"organizationRole" ASC, LOWER(users."lastName") ASC, LOWER(users."firstName") ASC');
+
+  const result = await fetchPage(queryBuilder, { number: pageNumber, size: pageSize });
+  const memberships = result.results.map(
+    (membership) =>
+      new Membership({
+        id: membership.membershipId,
+        organizationRole: membership.organizationRole,
+        updatedByUserId: membership.updatedByUserId,
+        user: new User(membership),
+      }),
+  );
+
+  return { models: memberships, pagination: result.pagination };
 };
 
-const findByUserIdAndOrganizationId = function ({ userId, organizationId, includeOrganization = false }) {
-  return BookshelfMembership.where({ userId, organizationId, disabledAt: null })
-    .fetchAll({ withRelated: includeOrganization ? ['organization', 'organization.tags'] : [] })
-    .then((memberships) => bookshelfToDomainConverter.buildDomainObjects(BookshelfMembership, memberships));
+export const findByUserIdAndOrganizationId = async ({ userId, organizationId, includeOrganization = false }) => {
+  const knexConnection = DomainTransaction.getConnection();
+  const memberships = await knexConnection(MEMBERSHIPS_TABLE).where({ userId, organizationId, disabledAt: null });
+
+  if (!includeOrganization) {
+    return memberships.map(toDomain);
+  }
+  const membershipOrganizationIds = memberships.map(({ organizationId }) => organizationId);
+  const organizations = await knexConnection(ORGANIZATIONS_TABLE).whereIn('id', membershipOrganizationIds);
+  const organizationIds = organizations.map(({ id }) => id);
+  const organizationsTags = await knexConnection(ORGANIZATION_TAGS_TABLE).whereIn('organizationId', organizationIds);
+
+  return memberships.map((membership) => {
+    const organization = organizations.find(({ id }) => id === membership.organizationId);
+    const organizationTags = organizationsTags.filter(({ organizationId }) => organizationId === organization.id);
+    return toDomain(membership, null, organization, organizationTags);
+  });
 };
 
-const updateById = async function ({ id, membership }) {
-  let updatedMembership;
+export const updateById = async ({ id, membership }) => {
+  const knexConnection = DomainTransaction.getConnection();
 
   if (!membership) {
     throw new MembershipUpdateError("Le membership n'est pas renseigné");
   }
-
-  try {
-    updatedMembership = await new BookshelfMembership({ id }).save(membership, {
-      patch: true,
-      method: 'update',
-      require: true,
-    });
-  } catch (err) {
-    throw new MembershipUpdateError(err.message);
+  if (!(await new knexConnection(MEMBERSHIPS_TABLE).select('id').where({ id }).first())) {
+    throw new MembershipUpdateError();
   }
 
-  const updatedMembershipWithUserAndOrganization = await updatedMembership.refresh({
-    withRelated: ['user', 'organization'],
+  await new knexConnection(MEMBERSHIPS_TABLE).where({ id }).update({
+    ...membership,
+    updatedAt: new Date(),
   });
```

Comment:
L'instruction `new` n'est pas nécessaire.
```suggestion
  await knexConnection(MEMBERSHIPS_TABLE).where({ id }).update({
    ...membership,
    updatedAt: new Date(),
  });
```